spec/factories/public_keys_factory.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryGirl.define do
+  factory :public_key do
+    association :user, factory: :user
+    name { Faker::Cat.name }
+    key { Faker::Crypto.sha256 }
```

Comment:
I didn't find any good faker method, so I just used this one. I also thought about generating real ssh keys, but to do that conveniently, we would need to use the `netssh` gem, which seemed to be overkill for this.